test/src/main/java/org/apache/accumulo/test/AuditMessageIT.java

```diff
@@ -482,6 +482,10 @@ public void testDeniedAudits() throws AccumuloSecurityException, AccumuloExcepti
       auditConnector.tableOperations().deleteRows(OLD_TEST_TABLE_NAME, new Text("myRow"),
           new Text("myRow~"));
     } catch (AccumuloSecurityException ex) {}
+    try {
+      auditConnector.tableOperations().flush(OLD_TEST_TABLE_NAME, new Text("start"),
+              new Text("end"), false);
```

Comment:
Normally, a range would be specified so that the start sorts before the end, but 's' sorts after 'e'. You should change it to match `myRow` and `myRow~` like the previous line, instead, so that way this will still work, even if we later add additional argument checks to make sure the start sorts before the end.

Reply:
Funny thing is, that is actually what I had originally but I changed it to mimic the parameters names for whatever reason.